lib/transaction_search.rb

```diff
@@ -41,7 +41,10 @@ def define_search_method(new_method_name, old_method_name)
         do_search(@search_fields)
         add_optimizations
         sort_and_paginate
-        render :layout => @layout if @layout
+        respond_with(@order_details) do |format|
+          format.html { render layout: @layout if @layout }
+          format.csv { render text: Reports::AccountTransactionsReport.new(@order_details, date_range_field: @date_range_field).to_csv }
```

Comment:
Do we need to set any additional headers? See `CSVHelper#set_csv_headers`. On that note, it might be nice get rid of the hack in that file where we use `CSVHelper::CSV` in order to support multiple ruby 1.8.7 and 1.9.2+. That could be a separate PR, but I would support it as part of this ticket.